js/index.js

```diff
@@ -7,6 +7,7 @@ import("../pkg").then(async fluvioWasm => {
       Fluvio.setupDebugging(false, 'Debug');
       const fluvio = await Fluvio.connect("ws://localhost:3000");
       const admin = await fluvio.admin();
+      console.log(`Using Fluvio Cluster Version: ${fluvio.platformVersion()}`);
```

Comment:
Could you add this line someplace in the [`test` of `simple.js`](https://github.com/infinyon/fluvio-client-wasm/blob/3fc246ce9f0e5ca699ec0d337941717d144b7c02/tests/js/simple/simple.js#L25-L60).

Now that I read the [Development section of the `README`](https://github.com/infinyon/fluvio-client-wasm#development) I see why you put it here. We could probably add a bit saying that the integration tests live in [`tests/js/<test name>/<test name>.js`](https://github.com/infinyon/fluvio-client-wasm/tree/main/tests/js).

Reply:
Yeah! I used the development server to see how changes affect on realtime. I added a `tests` directory
instruction to the README so it's more explicit as you suggest.